crypto/cms/cms_smime.c

```diff
@@ -25,6 +25,9 @@ static BIO *cms_get_text_bio(BIO *out, unsigned int flags)
         rbio = BIO_new(BIO_s_null());
     else if (flags & CMS_TEXT) {
         rbio = BIO_new(BIO_s_mem());
+        if (rbio == NULL) {
+            return NULL;
+        }
```

Comment:
The code style to not include the {} for a single statement body:
```suggestion
        if (rbio == NULL)
            return NULL;
```

There are a lot of these in this PR.